ansibullbot/utils/shippable_api.py

```diff
@@ -138,7 +138,7 @@ def _load_cache_file(self, cfile):
 
     def _write_cache_file(self, cfile, data):
         with gzip.open(cfile, 'w') as f:
-            f.write(json.dumps(data))
+            f.write(to_bytes(json.dumps(data)))
```

Comment:
And maybe https://github.com/ansible/ansibullbot/blob/master/ansibullbot/_json_compat.py#L38 ?